### PR TITLE
Remove unnecessary return statements in package

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR removes unnecessary `return` statements from the `chaco` package.

### CODE REVIEW

File: chaco/multi_line_plot.py [hunk 1]
```diff
@@ -443,28 +439,23 @@ def _render_icon(self, gc, x, y, width, height):
...
 
     def _line_style_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
 
     def _line_width_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
```
Comment (aaronayres35) [inline chaco/multi_line_plot.py:461]: unrelated to this PR but looks like these could all use the new `requires_redraw` metadata

**Review by aaronayres35 — APPROVED** [review]
LGTM

File: chaco/multi_line_plot.py (hunk 1 above)
Comment (rahulporuri) [reply chaco/multi_line_plot.py:461]: can you please create a general purpose issue to search and see where all we could use `requires_redraw` in chaco?

Comment (aaronayres35) [reply chaco/multi_line_plot.py:461]: I've opened #595

**Pull request merged**